partiql-parser/src/test/kotlin/org/partiql/parser/internal/PartiQLParserDDLTests.kt

```diff
@@ -371,7 +555,69 @@ class PartiQLParserDDLTests {
                         CHECK (SELECT a FROM foo)
                     )
                 """.trimIndent()
-            )
+            ),
+            ErrorTestCase(
+                "NULL not allowed as type in type decalration",
+                """
+                    CREATE TABLE TBL(
+                        a NULL
+                    )
+                """.trimIndent()
+            ),
+            ErrorTestCase(
+                "MISSING not allowed as type in type decalration",
+                """
+                    CREATE TABLE TBL(
+                        a MISSING
+                    )
+                """.trimIndent()
+            ),
+            ErrorTestCase(
+                "STRUCT<> NOT Supported",
+                """
+                    CREATE TABLE TBL(
+                        a STRUCT<>
+                    )
+                """.trimIndent()
+            ),
+            ErrorTestCase(
+                "LIST<> NOT Supported",
+                """
+                    CREATE TABLE TBL(
+                        a LIST<>
+                    )
+                """.trimIndent()
+            ),
+
+            // TODO: Fix Me
+            //  Potentially modify the bag grammar to use
+            //  ANGLE_LEFT ANGLE_LEFT ( expr ( COMMA expr )* )? ANGLE_RIGHT ANGLE_RIGHT
+            //  We should look into the multi-character token in ANTLR grammar
```

Comment:
Could cut an issue if we're not tackling it in the short-term

Reply:
Issue is fixed in the new commit.